esrally/driver/runner.py

```diff
@@ -1370,6 +1370,7 @@ async def __call__(self, es, params):
         try:
             for index_name in indices:
                 if not only_if_exists:
+                    request_params["ignore_unavailable"] = "true"
```

Comment:
I added this line to the `DeleteIndex` operation to avoid a failure...

Reply:
We will prefer handling via #1580 as it doesn't change client/operation behavior